src/stylelint-config.js

```diff
@@ -68,13 +66,11 @@ var configs = {
     "number-zero-length-no-unit": true,
     "property-case": "lower",
     "property-value-whitelist": {
-      "color": ["/^$/"]
+      "color": ["/(\$|\#)/"]
```

Comment:
I dont think you _need_ the '\' in front of the hastag but I don't think it hurts, either.